landingzones/lz-machinelearning/lz.bicep

```diff
@@ -383,6 +390,37 @@ module aksCluster '../../azresources/containers/aks/main.bicep' = {
   }
 }
 
+module appServicePlan '../../azresources/compute/web/app-service-plan-linux.bicep' = if (appservice.enabled) {
+  name: 'deploy-app-service-plan'
+  scope: rgCompute
+  params: {
+    name: appServicePlanName
+    skuName: 'P1V3'
```

Comment:
Let's add this to parameters.

Reply:
Added sku name and tier as parameters in the app service linux container object